pex/bin/pex.py

```diff
@@ -822,19 +827,19 @@ def to_python_interpreter(full_path_or_basename):
 
     pex_builder = PEXBuilder(path=safe_mkdtemp(), interpreter=interpreter, preamble=preamble)
 
-    def walk_and_do(fn, src_dir):
-        src_dir = os.path.normpath(src_dir)
-        for root, dirs, files in os.walk(src_dir):
+    if options.resources_directory:
+        pex_warnings.warn(
+            "The `-R/--resources-directory` option is deprecated. Resources should be added via "
+            "`-D/--sources-directory` instead."
```

Comment:
While I know we don't yet have a deprecation policy, thoughts on choosing a removal version here so that users can plan? Perhaps the version after this next one.

Reply:
Until we have a policy / infra I'd prefer not to defacto establish one ad-hoc.